deploy/assisted-service.yaml

```diff
@@ -102,6 +102,8 @@ spec:
               value: "false"
             - name: LOG_LEVEL
               value: "info"
+            - name: LOG_FORMAT
+              value: "json"
```

Comment:
Default for dev env can be text instead of json

Reply:
what is dev?